model-optimizer/telemetry/utils/sender.py

```diff
@@ -34,13 +34,16 @@ def _future_callback(future):
             with self._lock:
                 self.queue_size -= 1
 
+        callback_needed = False
         with self._lock:
             if self.queue_size < MAX_QUEUE_SIZE:
                 fut = self.executor.submit(backend.send, message)
```

Comment:
should it be also moved under if callback_needed?

Reply:
The code has been updated. The comment is not actual anymore